### PR TITLE
Additional permission and limit resources

### PR DESCRIPTION
**Pull request by paurosello**

### CODE REVIEW

File: aws-operator-role/iam-policy.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
+		        "iam:DeleteOpenIDConnectProvider",
                 "iam:ListOpenIDConnectProviderTags",
                 "iam:TagOpenIDConnectProvider",
                 "iam:UntagOpenIDConnectProvider",
                 "iam:ListOpenIDConnectProviders",
                 "iam:GetOpenIDConnectProvider",
-		"iam:UpdateOpenIDConnectProviderThumbprint",
-		"iam:RemoveClientIDFromOpenIDConnectProvider",
-		"iam:AddClientIDToOpenIDConnectProvider",
+		        "iam:UpdateOpenIDConnectProviderThumbprint",
```
Comment (whites11) [inline aws-operator-role/iam-policy.json:21]: seems wrong indentation

Comment (paurosello) [reply aws-operator-role/iam-policy.json]: yeah, just fixed, there were some tabs in there

**Review by T-Kukawka — APPROVED** [review]

File: aws-operator-role/iam-policy.json [hunk 2]
```diff
@@ -160,7 +159,7 @@
             ],
             "Resource": "${arn_prefix}:iam::${account_id}:role/aws-service-role/*"
         },
-	{
+    {
```
Comment (whites11) [inline aws-operator-role/iam-policy.json]: this was correct :)

Comment (paurosello) [reply aws-operator-role/iam-policy.json]: image
I think this is an issue in the PR preview, I can see it fine on my editor and github online editor

**Review by whites11 — APPROVED** [review]

**Pull request merged**